drumkit/index.html

```diff
@@ -12,15 +12,16 @@
 
   <h1 id="title">Drum 🥁 Kit</h1>
   <div class="set">
-    <button class="w drum">w</button>
+    <button class="w drum" >w</button>
```

Comment:
Scope of this PR is weather app only. So make sure the other things wont pushed together in this PR
Check other folder as well

Reply:
<img width="410" alt="Screenshot 2023-04-02 at 15 54 28" src="https://user-images.githubusercontent.com/58681637/229342835-141ee628-24ab-48a2-972a-f84c4065f0e9.png">
What are these folder????
Dont push node-modules folder